zerver/migrations/0465_rename_followup_day_email_templates.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.2.2 on 2023-07-11 10:45
+
+import orjson
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+WELCOME = 1
+
+
+def update_for_followup_day_email_templates_rename(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    ScheduledEmail = apps.get_model("zerver", "ScheduledEmail")
+    scheduled_onboarding_emails = ScheduledEmail.objects.filter(type=WELCOME)
```

Comment:
The furthest in the future that a followup_day2 email is scheduled is 3 days 23 hours from the current time, so we could exclude those from this query ...

```python
limit_for_followup_day2_emails = timezone_now() + datetime.timedelta(days=4)
...
scheduled_onboarding_emails = ScheduledEmail.objects.filter(type=WELCOME).exclude(
    scheduled_timestamp__gt=limit_for_followup_day2_emails
)
```

Reply:
I don't think this table is big enough to merit that optimization, and it has the potential to maybe be subtly wrong.

Reply:
Yeah, I had the same thought.